packages/terra-paginator/src/_paginationUtils.jsx

```diff
@@ -0,0 +1,43 @@
+const calculatePages = (totalCount, itemCountPerPage) => Math.ceil(totalCount / itemCountPerPage);
+
+const pageSet = (index, totalPages) => {
+  const sequence = [];
+  const numberShiftPoint = 7;
+  const maxPagesDisplayed = 10;
+  const previousSequenceCount = 5;
+  const nextSequenceCount = 4;
+  let mutableIndex = index;
```

Comment:
Would this potentially make more sense if we worded it as currentPage?

Reply:
Maybe? I think index/mutableIndex conveys what it is properly, though. Position in a set. Position in a set that has some processing applied.